app/pods/publications/index/route.js

```diff
@@ -80,6 +80,7 @@ export default class PublicationsIndexRoute extends Route {
         'numac-numbers',
         'publication-subcase',
         'translation-subcase',
+        'translation-subcase.request-activities',
```

Comment:
Deze `include` zou ik weglaten. Het bespaart een network request voor iedere rij, maar het verhoogt ook enorm de kans dat de cache entry snel geinvalideerd geraakt. Iedere creatie/wijziging van een request-activity gaat de hele lijst van publicaties invalideren in de cache waardoor de cache bijna nooit nog gehit zal worden voor het overzicht van publicaties (die redenering geldt trouwens ook voor nog andere velden in deze include denk ik).

In het algemeen heb ik het gevoel dat we nu soms te veel doorslaan richting `include` vs aantal network requests. In theorie besparen we op aantal requests, maar wanneer snel wijzigende data geinclude wordt, gaat dat resulteren in heel veel cache invalidations en in de praktijk dus toch veel langere load times. Dat is wat ik nu toch ervaar, ook al voor deze change, op het publicatie-overzicht.

Op sommige plaatsen, zoals bijvoorbeeld het agenda-overzicht, bespaart het effectief heel veel extra calls omdat er daar tot meer dan 100 agendaitems getoond kunnen worden waarvoor dan extra requests moeten gebeuren. Maar hier is de default page size 10 dus blijft het aantal extra requests sowieso beperkt. Ik weet dat de gebruiker de page size kan ophogen naar 100, maar op dat moment is het een bewuste keuze van de gebruiker om meer data te gaan tonen en lijkt het mij meer aanvaardbaar dat dat veel requests kan vragen.

Wat denken jullie @MikiDi @ValenberghsSven ?

Reply:
PS. de ideale oplossing is hier minder agressieve cache clearing, maar dat is niet voor direct, vrees ik :slightly_smiling_face: